app/forms/contact_detail_form.rb

```diff
@@ -70,4 +82,8 @@ def uk_address_must_not_be_empty
       )
     end
   end
+
+  def strip_email
```

Comment:
Would `sanitise_email` be a better description of behaviour?

Reply:
Much better, thanks. Updated!